docs/INSTALL_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+
+Binary downloads for a wide range of architectures and operating systems are
+available from the [InfluxData downloads][] page or from the
+[GitHub Releases][] page.
+
+[InfluxData downloads]: https://www.influxdata.com/downloads
+[GitHub Releases]: https://github.com/influxdata/telegraf/releases
+
+## Homebrew
+
+A [Homebrew Formulae][] for Telegraf that updates after each release:
+
+```shell
+brew update
+brew install telegraf
+```
+
+Note that the Homebrew organization builds Telegraf itself and does not use
+binaries built by InfluxData. This is important as Homebrew builds with CGO,
+which means there are some differences between the official binaries and those
+found with Homebrew.
+
+[homebrew formulae]: https://formulae.brew.sh/formula/telegraf
+
+## InfluxData Linux package repository
+
+InfluxData provides a package repo that contains both DEB and RPM packages.
+
+### DEB
+
+For DEB-based platforms (e.g. Ubuntu and Debian) run the following to add the
+repo GPG key and setup a new sources.list entry:
+
+```shell
+# influxdata-archive_compat.key GPG fingerprint:
+#     9D53 9D90 D332 8DC7 D6C8 D3B9 D8FF 8E1F 7DF8 B07E
+wget -q https://repos.influxdata.com/influxdata-archive_compat.key
+echo '393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c influxdata-archive_compat.key' | sha256sum -c && cat influxdata-archive_compat.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null
+echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list
+sudo apt-get update && sudo apt-get install telegraf
+```
+
+### RPM
+
+For RPM-based platforms (e.g. RHEL, CentOS) use the following to create a repo
+file and install telegraf:
+
+```shell
+# influxdata-archive_compat.key GPG fingerprint:
+#     9D53 9D90 D332 8DC7 D6C8 D3B9 D8FF 8E1F 7DF8 B07E
+cat <<EOF | sudo tee /etc/yum.repos.d/influxdata.repo
+[influxdata]
+name = InfluxData Repository - Stable
+baseurl = https://repos.influxdata.com/stable/\$basearch/main
+enabled = 1
+gpgcheck = 1
+gpgkey = https://repos.influxdata.com/influxdata-archive_compat.key
+EOF
+sudo yum install telegraf
+```
+
+## Official Docker images
+
+Telegraf is available as an [Official image][] on DockerHub. Official images
+are a curated set of Docker Images that also automatically get security updates
+from Docker, follow a set of best practices, and are available via a shortcut
+syntax which omits the organization.
+
+InfluxData maintains a Debian and Alpine based image across the last three
+minor releases. To pull the latest Telegraf images:
+
+```shell
+# latest Debian-based image
+docker pull telegraf
+# latest Alpine-based image
+docker pull telegraf:alpine
+```
+
+See the [Telegraf DockerHub][] page for complete details on available images,
+versions, and tags.
+
+[official image]: https://docs.docker.com/trusted-content/official-images/
```

Comment:
Is the capitalization important here for markdown linking?

Reply:
My understanding is that footnotes are case-insensitive